src/pluto/components/corrections_report.py

```diff
@@ -0,0 +1,141 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
```

Comment:
What does this do?

Reply:
it stand for "abstract base class", which is a base class that can't be instantiated. The only objects created with the implemented functionality are from classes that inherit from the abc.

Reply:
👍 makes sense